ecosystem/lava-sdk/src/rpcconsumer/rpcconsumer_server.ts

```diff
@@ -90,7 +90,8 @@ export class RPCConsumerServer {
   }
 
   async sendRelay(
-    options: SendRelayOptions | SendRelaysBatchOptions | SendRestRelayOptions
+    options: SendRelayOptions | SendRelaysBatchOptions | SendRestRelayOptions,
+    virtualEpoch: number
```

Comment:
let's align the code with the go implementation, we are fetching the virtual epoch inside the rpcconsumer_server no reason to do it differently and propagate it in all our functions, just bring a wrapper that can get the virtual epoch to the class and call it before getSessions like the go code